locales/locale_fr.po

```diff
@@ -25,6 +22,30 @@ msgstr "Volume"
 msgid "Back"
 msgstr "Retour"
 
+msgid "Music"
+msgstr "Musique"
+
+msgid "Mute/Unmute"
+msgstr "Muet/Non-muet"
+
+msgid "Sfx"
+msgstr "Effets sonores"
+
+msgid "Language"
+msgstr "Langue"
+
+msgid "English"
+msgstr "Anglais"
```

Comment:
The language names should appear in their original names.

For example, imagine if we were to add Chinese support, a non-Chinese speaking user would not be able to know which language is which if they switch to Chinese.